src/components/pages/novu-framework/code/code.jsx

```diff
@@ -0,0 +1,148 @@
+import clsx from 'clsx';
+import { StaticImage } from 'gatsby-plugin-image';
+import React, { useState } from 'react';
+import { PrismLight as SyntaxHighlighter } from 'react-syntax-highlighter';
+import javascript from 'react-syntax-highlighter/dist/esm/languages/prism/javascript';
+
+import createCustomElement from './create-custom-element';
+
+const CODE = `import { Echo } from '@novu/echo';
+
+const echo = new @@Echo@echoTooltip@@();
+
+const commentWorkflow = echo.workflow('comment-on-post', async (@@event@eventTooltip@@) => {
+  const inAppResponse = await event.step.inApp('notify-user', async () => ({
+    body: renderReactComponent(event.@@payload@payloadTooltip@@.postId)
+  }));
```

Comment:
All of this still uses the old way. Should we rewrite to use the new way?

Reply:
Same comment as above. :)